backend/src/main/java/hanglog/trip/presentation/CityController.java

```diff
@@ -0,0 +1,24 @@
+package hanglog.trip.presentation;
+
+import hanglog.trip.dto.response.CityResponse;
+import hanglog.trip.service.CityService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/cities")
+public class CityController {
+
+    private final CityService cityService;
+
+    @GetMapping
+    public ResponseEntity<List<CityResponse>> getCities() {
+        return ResponseEntity.ok(cityService.getAllCities());
```

Comment:
```suggestion
        final List<CityResponse> cityResponses = cityService.getAllCities();
        return ResponseEntity.ok(cityResponses);
```
다른 컨트롤러들은 분리해두었길래....

Reply:
[refactor: 서비스 변수 추출](https://github.com/woowacourse-teams/2023-hang-log/pull/105/commits/ce555d1db7eb4ba1ba92fce5f08dcb57a8137f2d)